### PR TITLE
Added the ESP Registry at the project highlight section

### PR DESCRIPTION
**Pull request by pedrominatel**
- Added the new section: Project Highlight
- Added the ESP Registry at the Project Highlight

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -7,6 +7,18 @@ To check out all the series of SoCs from Espressif, please visit our [ESP Produc
...
+<div align="center">
+    <img src="profile/esp_registry_logo.png">
+</div>
+
+###  **ESP Registry**
+
+The **ESP Registry** is a tool that downloads dependencies for any ESP-IDF CMake project. It makes sure that the right versions of all components required for a successful build of your project are in place. The download happens automatically during a run of CMake.
+
+Find the most exciting ESP-IDF components on [ESP Registry](https://components.espressif.com/)!
+
```
Comment (VojtechBartoska) [inline profile/README.md:21]: _What about adding link to Component Manager repository for contributions?_

```suggestion
If you want to contribute to this project, you can do so at [IDF Component Manager repository](https://github.com/espressif/idf-component-manager).
```

**Review by VojtechBartoska — APPROVED** [review]
LGTM, one small suggestion.

**Comment by pedrominatel** [comment]
PTAL @igrr

File: profile/README.md [hunk 2]
```diff
@@ -7,6 +7,18 @@ To check out all the series of SoCs from Espressif, please visit our [ESP Produc
...
+<div align="center">
+    <img src="profile/esp_registry_logo.png">
+</div>
+
+###  **ESP Registry**
+
+The **ESP Registry** is a tool that downloads dependencies for any ESP-IDF CMake project. It makes sure that the right versions of all components required for a successful build of your project are in place. The download happens automatically during a run of CMake.
+
+Find the most exciting ESP-IDF components on [ESP Registry](https://components.espressif.com/)!
+If you want to contribute to this project, you can do so at [IDF Component Manager repository](https://github.com/espressif/idf-component-manager).
```
Comment (igrr) [inline profile/README.md:21]: I'm not sure if a lot of meaningful contributions can be done inside idf-component-manager repo, it's mostly the tooling part. How about we recommend the users to publish their components in the Registry, instead?

Comment (VojtechBartoska) [reply profile/README.md:21]: Good point, I was mainly thinking about the feedback which should be shared in the idf-component-manager repository.

**Comment by VojtechBartoska** [comment]
cc [@kumakey](https://github.com/kumekay)

**Review by igrr — APPROVED** [review]

**Comment by VojtechBartoska** [comment]
@pedrominatel it looks this PR was forgotten, can we merge it? :)

**Pull request merged**